### PR TITLE
set up the project refs #1

### PR DESCRIPTION
**Pull request by heron2014**
## Ready to :eyes: 

Related to #1

Thanks @minaorangina :tulip:

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-1"]
```
Comment (minaorangina) [inline .babelrc]: Are you using the Babel stage 1 preset for any particular reason?  Stage 0/1/x presets support features that are currently being _proposed_ for future versions of JavaScript.  In fact, many features in Stage X plugins might disappear and never make the next official version of JavaScript at all.

They rank from 0 - 4, with stage 0 being literally super early stage proposals, and 4 being features that will be finalised eventually.

Basically, unless there's some specific experimental feature you specifically want to use, you don't need it.  You can [read about it here](http://babeljs.io/docs/plugins/)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "jquery": "^3.1.0",
+    "jsdom": "^9.5.0",
+    "mocha": "^3.0.2",
+    "react-addons-test-utils": "^15.3.2",
+    "webpack": "^1.13.2",
+    "webpack-dev-server": "^1.16.1"
+  },
+  "dependencies": {
+    "babel-preset-stage-1": "^6.13.0",
+    "eslint": "^3.5.0",
```
Comment (minaorangina) [inline package.json]: I would put eslint in the dev dependencies

File: package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    "chai": "^3.5.0",
+    "chai-jquery": "^2.0.0",
+    "eslint-plugin-react": "^6.3.0",
+    "ghooks": "^1.3.2",
+    "istanbul": "^1.0.0-alpha.2",
+    "jquery": "^3.1.0",
+    "jsdom": "^9.5.0",
+    "mocha": "^3.0.2",
+    "react-addons-test-utils": "^15.3.2",
+    "webpack": "^1.13.2",
```
Comment (minaorangina) [inline package.json]: webpack needs to be in the dependencies, otherwise heroku can't build your project

File: package.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    "url": "https://github.com/brainsandbrawns/client/issues"
+  },
+  "config": {
+    "ghooks": {
+      "pre-commit": "npm run test"
+    }
+  },
+  "homepage": "https://github.com/brainsandbrawns/client#readme",
+  "devDependencies": {
+    "babel-core": "^6.14.0",
```
Comment (minaorangina) [inline package.json]: all the babel stuff needs to be in the dependencies, because webpack uses them.  See my comment on webpack

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    path: require('path').join(__dirname, 'public'),
+    publicPath: '/',
+    filename: 'bundle.js'
+  },
+  module: {
+    loaders: [{
+      exclude: /node_modules/,
+      loader: 'babel',
+      query: {
+        presets: ['react', 'es2015', 'stage-1']
```
Comment (minaorangina) [inline webpack.config.js]: Here, you're configuring Babel to work with the Babel presets you want in the `query` object.  But you're also doing the exact same thing in your `babelrc`.  

You can define your presets for Babel either here or in a `babelrc` file, but you shouldn't do both - you're just doing it twice.

File: package.json (hunk 2 above)
Comment (heron2014) [inline package.json]: Done!

File: package.json (hunk 3 above)
Comment (heron2014) [inline package.json]: Done!

File: webpack.config.js (hunk 5 above)
Comment (heron2014) [inline webpack.config.js]: You right I am doing this twice! I removed presets from webpack in a favour of babelrc

File: package.json (hunk 4 above)
Comment (heron2014) [inline package.json]: done!

File: .babelrc (hunk 1 above)
Comment (heron2014) [inline .babelrc]: To be honest this confuses me! I installed babel-preset-0 which include all other presets possibilities.

**Comment by heron2014** [comment]
@minaorangina I made recommended changes. Have a look please :) Thanks!

**Comment by heron2014** [comment]
I made massive refactor, and changes moved to another PR - https://github.com/brainsandbrawns/client/pull/6
